html/captive-portal/templates/emails/emails-guest_password_of_the_day.mjml

```diff
@@ -0,0 +1,32 @@
+<mjml>
+  <mj-head>
+    <mj-include path="_header.mjml" />
+  </mj-head>
+  <mj-body>
+    <mj-section></mj-section>
+    <mj-wrapper>
+      <mj-section>
+        <mj-column>
+          <mj-text mj-class="title">[% i18n("Password Of the Day") %]</mj-text>
+        </mj-column>
+      </mj-section>
+      <mj-section>
+        <mj-column>
+          <mj-divider></mj-divider>
+        </mj-column>
+      </mj-section>
+      <mj-section padding-top="0">
+        <mj-column>
+          <mj-text>
+            <p>[% i18n("Hello") %],</p>
+            <p>[% i18n("Here the new password updated for the Password Of the Day") %]</p>
+            <p>[% i18n("Here are the info.") %]</p>
+            8n("Username") %]: [% pid %]</br> [% i18n("Password") %]: [% password %]</br> [% i18n("Expire") %]: [% expiration %]</p>
```

Comment:
Not sure if you tested this, but the username i18n call isn't valid and the TT brackets aren't balanced so I'm not even sure this will compile